package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Web Version",
   "scripts": {
     "build": "cross-env NODE_ENV=production webpack",
-    "dev": "cross-env NODE_ENV=development webpack serve --inline --open --mode=development",
```

Comment:
The `--inline` option was removed. From what I've read it's inherently true now.